build/sections/skills.html

```diff
@@ -0,0 +1,47 @@
+<div class="container pb-10">
+    <div class="text-xl font-raleway-Bold lg:text-[26px] pt-5 xl:pt-8 text-white">
+        <h1>Eine mögliche T<span class="before-underline">eamkonstellation</h1></span>
+    </div>
+    <div class="pt-36 grid grid-cols-1 place-items-center gap-6 text-white md:hidden lg:hidden pb-32">
+        <div class="relative ">
+            <img src="Images/Arnold.webp" alt="Arnold">
+            <div class="bg-[url('../Images/bgteam.webp')] absolute bottom-0 px-4">
+                <h4 class="font-raleway-Bold text-2xl py-2">Arnod:</h4>
+                <p class="text-base font-montserrat-Medium italic py-2">“d diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam“</p> 
+            </div>
+        </div>
+        <div class="relative items-center">
+            <img src="Images/Clever.webp" alt="Clever">
+            <div class="bg-[url('../Images/bgteam.webp')] absolute bottom-0 px-4">
+                <h4 class="text-2xl font-raleway-Bold py-2">Clever:</h4>
+                <p class="text-base font-montserrat-Medium italic py-2">“d diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam“</p> 
+            </div>
+        </div>
+        <div class="relative items-center">
+            <img src="Images/Izzeddin.webp" alt="Izzeddin">
+            <div class="bg-[url('../Images/bgteam.webp')] absolute bottom-0 px-4">
+                <h4 class="text-2xl font-raleway-Bold py-2">Izzeddin:</h4>
+                <p class="text-base font-montserrat-Medium italic py-2">“d diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam“</p> 
+            </div>
+        </div>
+    </div>
+    <div class="py-40 md:relative md:flex lg:relative lg:flex md:justify-center lg:justify-center md:pb-28 lg:pb-12 hidden">
+        <img src="Images/Pictures.webp" class="md:w-[100%] lg:w-[80%]" alt="Team">
```

Comment:
You might need to look into this again.
```suggestion
        <img src="Images/Pictures.webp" class="md:w-full lg:w-[80%]" alt="Team">
```